setup.py

```diff
@@ -54,7 +54,9 @@ def read_version_py(infname):
     # * foolscap >= 0.12.5 has ConnectionInfo and ReconnectionInfo
     # * foolscap >= 0.12.6 has an i2p.sam_endpoint() that takes kwargs
     # * foolscap 0.13.2 drops i2p support completely
-    "foolscap == 0.13.1",
+    # * foolscap >= 20.4 is necessary for Python 3
+    "foolscap == 0.13.1; ; python_version < '3.0'",
+    "foolscap >= 20.4.0; ; python_version > '3.0'",
```

Comment:
I believe there's an [extra](https://www.python.org/dev/peps/pep-0508/#environment-markers) semicolon here? I rebased #732 on this branch and am seeing this following behavior (with one semicolon I don't hit this error):

```
$ tox -e py38
GLOB sdist-make: /Volumes/workbench/snakedev/tahoe/tahoe-lafs/setup.py
ERROR: invocation failed (exit code 1), logfile: /Volumes/workbench/snakedev/tahoe/tahoe-lafs/.tox/log/GLOB-0.log
====================================================== log start =======================================================
error in tahoe-lafs setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Invalid requirement, parse error at "'; ; pyth'"

======================================================= log end ========================================================
ERROR: FAIL could not package project - v = InvocationError(u'/Users/whit537/.asdf/installs/python/2.7.18/bin/python2.7 setup.py sdist --formats=zip --dist-dir .tox/dist', 1)
$
```